doc/src/vpr/command_line_usage.rst

```diff
@@ -1254,6 +1254,14 @@ Analysis Options
 
     **Default:** ``off``
 
+.. option:: --gen_post_synthesis_merged_netlist { on | off }
+
+    This option is based on ``--gen_post_synthesis_netlist``.
```

Comment:
typo: "*a* single verilog file"
I think we should change the name to --gen_post_implementation_netlist, as @acomodi suggests.

Reply:
Fixed typo and renamed option